libbeat/autodiscover/providers/kubernetes/pod.go

```diff
@@ -356,10 +356,7 @@ func (p *pod) emitEvents(pod *kubernetes.Pod, flag string, containers []kubernet
 		)
 
 		cmeta := common.MapStr{
-			"id": cid,
-			"image": common.MapStr{
-				"name": c.Image,
-			},
```

Comment:
This seems to be removing fields from the `container` object in meta, shouldn't we be removing the fields from `kubemeta["container"]`, so they are removed from `kubernetes.container` object in meta?

Reply:
well If i'm not mistaken `cmeta` is the final object that will be used to enrich the events so we explicitly add `id` and `runtime` here while the container.image.name and container.name will be added through the alias.

`kubemeta` on the other hand contains the data that will be used to evaluate configurations like in a condition: 
```
contains: kubernetes.container.image: redis
```

Does this make sense?

Reply:
Oh ok, makes sense, I always get confused about these metas.